.github/workflows/verify_failure.yml

```
@@ -0,0 +1,55 @@
+            if ./build/Example; then
+              echo "ERROR: Test should have failed but passed"
+              exit 1
+            else
+              echo "Test failed as expected"
+              exit 0
+            fi
+
+  test-emscripten:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.8)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)